setup.sh

```diff
@@ -0,0 +1,100 @@
+#/usr/bin/env bash
+
+test -r ~/.dotfiles_initialized && echo "dotfiles already setup; ~/.dotfiles_initialized exists!" && exit 0
+
+read -p "Setup requires an internet connection, is long, and requires checkins. Is this okay? (Y/n)? " answer
+case ${answer:0:1} in
+    y|Y )
+    ;;
+    * )
+	    exit 0
+    ;;
+esac
+
+set -x
+find ~ -mindepth 1 -maxdepth 1 -type d -exec rmdir {} \;
```

Comment:
`find` is such an interesting command. With my recent experience I would have written this:
```
ls -a $HOME | xarg rmdir
```
I know this will try and delete all files, but `rmdir` will not delete them so this is fine. 

I wanna do benchmarking but that is silly lol.